exchanges/bybit/bybit_types.go

```diff
@@ -907,7 +907,7 @@ type WalletBalance struct {
 	} `json:"list"`
 }
 
-// UnifiedAccountUpgradeResponse represents a response parameter for update to unified account.
+// UnifiedAccountUpgradeResponse represents a response parameter for update to unified accounts.
```

Comment:
I think I will stop mentioning the reverts now but there is more.

Reply:
Fixed https://github.com/gbjk/gocryptotrader/commit/ec9604b1d

Pretty sure that's all of them.
Sorry, I thought I'd caught them all previously.

Reply:
Fixed https://github.com/gbjk/gocryptotrader/commit/0cd6dadb0

That one was a slippery bugger and nearly escaped the net.